src/csharp/GeneratorParams.cs

```diff
@@ -37,20 +36,19 @@ public void TryGraphCaptureWithMaxBatchSize(int maxBatchSize)
 
         public void SetInputIDs(ReadOnlySpan<int> inputIDs, ulong sequenceLength, ulong batchSize)
         {
-            unsafe
-            {
-                fixed (int* inputIDsPtr = inputIDs)
-                {
-                    Result.VerifySuccess(NativeMethods.OgaGeneratorParamsSetInputIDs(_generatorParamsHandle, inputIDsPtr, (UIntPtr)inputIDs.Length, (UIntPtr)sequenceLength, (UIntPtr)batchSize));
-                }
-            }
+            Result.VerifySuccess(NativeMethods.OgaGeneratorParamsSetInputIDs(_generatorParamsHandle, inputIDs.ToArray(), (UIntPtr)inputIDs.Length, (UIntPtr)sequenceLength, (UIntPtr)batchSize));
```

Comment:
I think we need not do this. `inputIDs.ToArray()` will make a copy of the underlying array if I remember correctly. Instead, we could just send in a pointer to the underlying memory buffer using the previous implementation.
For this function, maybe we can leave it unchanged.

Reply:
👍Reverted